keps/sig-cli/kustomize-secret-generator-plugins.md

```diff
@@ -179,12 +228,24 @@ func Env(root string, args []string) ([]kv.Pair, error) {
 }
 ```
 
-Users would install the plugin with `go build -buildmode=plugin ~/.config/kustomization_plugins/myplugin.so`
+Users would install the goplugin with:
+
+```
+go build -buildmode=plugin ~/.config/kustomize/plugins/kvSources/myplugin.so
+go get sigs.k8s.io/kubernetes-sigs/kustomize
+```
+
 
-### Risks and Mitigations
 
-#### Several people want an _exec-style_ plugin
-_exec-style_ means execute arbitrary code from some file.
+## Risks and Mitigations
+
+### Several people want an _exec-style_ plugin
+
+_exec-style_ means execute arbitrary code from some file,
```

Comment:
Note: This solves a different problem than the one in this KEP.  It will have a much broader application and wider user base.  As such it needs to be more carefully thought out w.r.t. the goals, and needs to have a stronger commitment towards supporting it with documentations, answering questions, debugging issues, etc.  The users for this are expected to have less knowledge about the workings of kustomize..  There are a number of issues that would need to be considered - such as would it use kubectl plugins, or its own kubectl-style-plugin with a different name prefix?  Do these need to be flag enabled, or should they be enabled by default?  Should they work against transitive bases, or just at the topmost layer.  Should these work on both remote and local targets, or only local targets?

Reply:
added commentary to this effect